dspace/config/modules/uclouvain.cfg

```diff
@@ -1,7 +1,8 @@
 uclouvain.main_dial_url = https://dial.uclouvain.be
 uclouvain.main_author_query.url = ${uclouvain.main_dial_url}/cgi-bin/valet_utils
 
-uclouvain.main_author_query.filter = student
+uclouvain.main_author_query.filter = student_ucl
+uclouvain.main_promoter_query.filter = academic
```

Comment:
It could be interresting to have comment to explain config possibilities (or where find possible values)